assets/webpack.config.js

```diff
@@ -56,6 +56,9 @@ module.exports = (env, options) => ({
   },
   plugins: [
     new MiniCssExtractPlugin({ filename: "../css/app.css" }),
-    new CopyWebpackPlugin([{ from: "static/", to: "../" }])
+    new CopyWebpackPlugin([
+      { from: "vendor/ex_admin", to: "../" },
```

Comment:
Strings must use singlequote.

[Read more about it here.](http://eslint.org/docs/rules/quotes)